### ISSUE TITLE
Photos stuck in processing since August.

### ISSUE THREAD

**Issue by james2432** [openstreetcam/openstreetview.org#155]
http://openstreetcam.org/details/512250
http://openstreetcam.org/details/512248
http://openstreetcam.org/details/512244

**Comment by ToeBee** [comment]
Yeah, I'm up to 27 stuck in processing. All 3 of my uploads today got stuck. What is really annoying me is that I can't delete tracks that are stuck in processing. There are several that are failed uploads and I would like to get rid of them.

**Comment by zstadler** [comment]
Some tracks are stuck in "processing" for more than a year.

Would it be possible to re-start processing of all unprocessed tracks that were uploaded more than 3 months ago?

**Comment by nakic** [comment]
It appears to me that this issue and #207 are the same, with #207 being a bit newer, with more activity. Shall we close this one so that people can follow events under #207?
